src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/common/SupportedAppointmentTypes.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.common
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.config.Feature
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.config.FeatureSwitches
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity.BookingHistoryAppointment
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity.PrisonAppointment
+
+@Component
+class SupportedAppointmentTypes(private val featureSwitches: FeatureSwitches) {
```

Comment:
An effort to centralise this logic.  In theory if we need to support 'other' types when the time comes we just add to here.

Normally the crystal ball eventually cracks somewhere down the line ...